lib/phoenix/live_dashboard/helpers.ex

```diff
@@ -222,4 +222,6 @@ defmodule Phoenix.LiveDashboard.Helpers do
   All connected nodes (including the current node).
   """
   def nodes(), do: [node()] ++ Node.list(:connected)
+
+  def unique_id, do: 16 |> :crypto.strong_rand_bytes() |> Base.url_encode64(padding: false)
```

Comment:
@josevalim I'm not really happy about this approach. Do you have a better idea?

Reply:
Yes, me neither. It will make everything considerably more expensive too. One option I had in mind would be to convert them `progress` but they are annoying to style across browsers.

Reply:
@josevalim That would work for the progress pattern specifically, but it would for sure not work with the memory usage bar and also not as a generic solution that allows css in pages.

We could just give them specific ids...?

Reply:
We could. We can probably generate IDs in all cases. Even for the core cases they are obviously sequential.

Reply:
@josevalim I'll have a look again if i find a nicer id solution then :+1:

Reply:
@josevalim I set reasonable ids without any randomness for all components